src/rajaperf_config.hpp.in

```diff
@@ -24,32 +24,35 @@
 
 #include <string>
 
+#cmakedefine RAJA_PERFSUITE_ENABLE_MPI
+
 namespace rajaperf {
 
 struct configuration {
-
+#if 0
```

Comment:
??

Reply:
This is here because what follows is not a valid way to initialize static strings in c++11. This file was not actually included before so it never broke anything.

Reply:
Okay, makes sense. I think Holger fixed this in his PR: #137